### PR TITLE
Fix nits in WebCodecs article

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR fixes some nits in the WebCodecs article:
- Prettier JS formatting
- Use `const` instead of `let` when appropriate in JS snippets
- Use camelCase instead of snake_case for variable names and functions in JS snippets 
- Remove extra declarations in JS snippets
- Replace `wicg` with `w3c` in links
- Fix not existing links
- And other nits/typos.

### CODE REVIEW

**Comment by chcunningham** [comment]
@Djuffin, can you take a look?

File: src/site/content/en/blog/webcodecs/index.md [hunk 1]
```diff
@@ -273,24 +270,24 @@ Setting up a `VideoDecoder` is similar to what's been done for the
 parameters are given to `configure()`.
 
 The set of codec parameters varies from codec to codec. For example H.264 codec
-might need a [binary blob](https://wicg.github.io/web-codecs/#dom-audiodecoderconfig-description)
-of avcC, unless it's encoded in so called AnnexB format ( `encoderConfig.avc = { format: "annexb" }` ).
+might need a [binary blob](https://w3c.github.io/webcodecs/#dom-videodecoderconfig-description)
+of AVVC, unless it's encoded in so called Annex B format (`encoderConfig.avc = { format: "annexb" }`).
```
Comment (Djuffin) [inline src/site/content/en/blog/webcodecs/index.md:274]: AVCC instead of AVVC

Comment (Djuffin) [reply src/site/content/en/blog/webcodecs/index.md:274]: LGTM

**Comment by Djuffin** [comment]
LGTM

File: src/site/content/en/blog/webcodecs/index.md (hunk 1 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/webcodecs/index.md:274]: Thanks for catching this typo!

**Pull request merged**